plugin/src/main/java/com/denizenscript/denizen/utilities/command/DenizenCommandHandler.java

```diff
@@ -23,6 +23,10 @@
 
 public class DenizenCommandHandler {
 
+    public static final String DISCORD_LINK = "https://discord.gg/Q6pZGSR";
+    public static final String WEBSITE_LINK = "https://denizenscript.com";
+    public static final String GUIDE_LINK = "https://guide.denizenscript.com/";

```

Comment:
This shouldn't be here

Reply:
I personally disagree, but I suppose I am following your formatting rules within this project, so I will revert the change.